src/coreclr/debug/daccess/daccess.cpp

```diff
@@ -5656,7 +5656,7 @@ static int FormatCLRStubName(
 
     // Compute the address as a string safely.
     WCHAR addrString[Max64BitHexString + 1];
-    FormatInteger(addrString, ARRAY_SIZE(addrString), "%p", stubAddr);
+    FormatInteger(addrString, ARRAY_SIZE(addrString), "%p", (ULONG64)stubAddr);
```

Comment:
ULONG64 for %p format spec does not look right.

Reply:
I will copy the error when I get back to my macOS machine. It fails the assert for integer types.

Reply:
osx-x64 build:
```
  In file included from <built-in>:490:
  In file included from /Users/filipnavara/Projects/runtime/artifacts/obj/coreclr/osx.x64.Debug/debug/daccess/CMakeFiles/daccess.dir/cmake_pch_x86_64.hxx:5:
  In file included from /Users/filipnavara/Projects/runtime/src/coreclr/debug/daccess/stdafx.h:42:
  In file included from /Users/filipnavara/Projects/runtime/src/coreclr/vm/common.h:94:
  In file included from /Users/filipnavara/Projects/runtime/src/coreclr/vm/util.hpp:14:
  /Users/filipnavara/Projects/runtime/src/coreclr/inc/utilcode.h:283:5: error: static assertion failed due to requirement 'std::is_integral<unsigned long>::value': Integral type required.
      static_assert(std::is_integral<I>::value, "Integral type required.");
      ^             ~~~~~~~~~~~~~~~~~~~~~~~~~~
  /Users/filipnavara/Projects/runtime/src/coreclr/debug/daccess/daccess.cpp:5659:5: note: in instantiation of function template specialization 'FormatInteger<unsigned long>' requested here
      FormatInteger(addrString, ARRAY_SIZE(addrString), "%p", stubAddr);
      ^
  1 error generated.
  make[2]: *** [debug/daccess/CMakeFiles/daccess.dir/daccess.cpp.o] Error 1
  make[1]: *** [debug/daccess/CMakeFiles/daccess.dir/all] Error 2
```

Reply:
I figured the right fix for this one.